### ISSUE TITLE
Add license information

### ISSUE THREAD

**Issue by vazhnov** [exentio/koyomi-lvds-hat#2]
Hello!

I found this project from https://hackaday.com/2024/05/14/pcb-design-review-hdmi-to-lvds-sony-vaio-lcd-devboard/

Could you please add information about the license of this repository — is it allowed to copy, change and use the code and PCB hardware files in any purpose?
It would be great to see any popular open source license.

**Comment by exentio** [comment]
I've been pondering about adding a license to all Koyomi-related repos lately, it's a bit late right now for me but tomorrow I'll start looking at which license would be better for the project! I want people to use and reference my designs freely and make their boards so I'll choose a very permissive one!

**Comment by vazhnov** [comment]
> I'll choose a very permissive one!

Sounds like MIT license :)
You can add it through GitHub templates: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository

**Comment by exentio** [comment]
I ended up choosing the strongly reciprocal CERN Open Hardware Licence because I believe that sharing every improvement to my designs would benefit everyone, especially those who want to reference my work and the improvements made by others for different projects.  
On various occasions during the development of the two LVDS boards I encountered projects (mostly by the same person) that tried to achieve similar goals but the authors never shared anything, despite putting the Creative Commons logo on their boards. Being able to look at the schematics would've made the development of these boards quite easier, so I want to be helpful to people in a situation like mine!
